docker-traceability-plugin/src/main/java/org/jenkinsci/plugins/docker/traceability/JenkinsInstance.java

```diff
@@ -0,0 +1,29 @@
+package org.jenkinsci.plugins.docker.traceability;
+
+import javax.annotation.CheckForNull;
+
+import jenkins.model.Jenkins;
+
+public class JenkinsInstance {
```

Comment:
Agreed, must have a `private` constructor but seems to me that `Utils` name is too generic given that the utility class only wraps the call to `Jenkins.getInstance()`.

May I suggest `JenkinsInstanceWrapper`?